DAG-update-elk-convcollective.py

```diff
@@ -1,128 +0,0 @@
-from datetime import timedelta
```

Comment:
Can we keep this file. Update in target architecture can be adapted but not suppressed entirely. I would like to have a backbone (like this one) to start reshaping it.

Reply:
Okay, done !